scripts/sqlqueries.sql

```diff
@@ -172,14 +217,15 @@ WHERE
 ORDER BY 
     s.name, 
     acc_cpi.year,
-    quarter;
+    quarter asc;

```

Comment:
I think we should do driverpop / 4, since cpi has quarterly data and not monthly